prov/util/src/util_mr_cache.c

```diff
@@ -388,7 +391,7 @@ int ofi_mr_cache_search(struct ofi_mr_cache *cache, const struct ofi_mr_info *in
 struct ofi_mr_entry *ofi_mr_cache_find(struct ofi_mr_cache *cache,
 				       const struct fi_mr_attr *attr)
 {
-	struct ofi_mr_info info;
+	struct ofi_mr_info info = {0};
```

Comment:
ofi_mr_info is over 100 bytes in size, and cache lookups are in the fast path.  I get it's safer to always clear the entire structure, but I'd prefer that we initialize only the fields that are needed.  We can include setting iface for future protection, since ipc and device are only defined if iface != system.